### ISSUE TITLE
Display of Power implementation

### ISSUE THREAD

**Issue by Legion310513** [gryffon/ringteki#872]
Hi, When playing Display of Power I successfully claim the ring, but when trying to resolve the elemental effects I click yes, but I do not get the option to actually resolve the.
My Jigoku profile name is Legion_310513 & I completed a game at approximately 21:40GMT on 07/11/2017.
The game host was Kharmasoup70.

Thanks for all your hard work.

**Comment by Arbuzio** [comment]
yes, DoP doesnt work on jigoku.online correclty - claimed the ring for me, but asked my opponent if he wants to use the ring
